sphinx-docs/installguide/install_mac.rst

```diff
@@ -3,23 +3,24 @@ Mac Installation Guide
 
 Macintosh installation uses the command line, but if you follow the instructions carefully, you should have no problem installing KA Lite. KA Lite uses the `git` program to distribute its software to developers and other users.
 
-#. Download and install either Python2.6_ or Python2.7_ for Mac
-	.. _Python2.6: https://www.python.org/download/releases/2.6/
-	.. _Python2.7: https://www.python.org/download/releases/2.7/
-#. You need to open the Terminal app. There are many ways to do this. We suggest the following:
+1. Download and install `Python 2.7.9 <https://www.python.org/downloads/mac-osx/>`_ for Mac
```

Comment:
Note: This won't be necessary once we have an OS X installer, as we have a python interpreter bundled in there.